content/docs/alerting/configuration.md

```diff
@@ -70,6 +70,8 @@ global:
   [ smtp_smarthost: <string> ]
   # The default hostname to identify to the SMTP server.
   [ smtp_hello: <string> | default = "localhost" ]
+  # Now alertmanager does not support NTLM authentication, if use SMTP server, please set it as "" or comement it to disable
+  # SMTP authentication.
```

Comment:
`SMTP username for CRAM-MD5, LOGIN and PLAIN. If empty, Alertmanager doesn't authenticate to the SMTP server.`

Reply:
To be consistent with the rest, it should be "SMTP Auth using CRAM-MD5, LOGIN and PLAIN."

Reply:
No NTLM? should it be `Alertmanager support SMTP Auth with  CRAM-MD5, LOGIN and PLAIN`?